back/db/views/analytics_build_feedbacks_v02.sql

```diff
@@ -18,7 +18,7 @@ FROM (
     UNION ALL
 
     SELECT
-        post_id,
+        idea_id AS post_id,
```

Comment:
Does editing a view SQL file like this result in the change being applied?

If 'yes', then why do we sometimes need to write a new version and apply it with a migration, but other times (like this) we can just edit the file?

Reply:
Maybe we choose to use a new file (with a new version in the name) so we have a record of changes made in case we need to revert and replace/convert related data structures.

In which case, I guess my question is, are we sure we don't need to keep such a record by using a new version file, rather than editing in place like this?

Reply:
Well spotted! This is a mistake, indeed, I should not have edited the view and should create a new one instead.

Reply:
@sebastienhoorens I only highlighted this one case, but there others in this PR where a view is edited and not replaced by a new version (+ migration to apply it).

Reply:
Alright, I'll double check all the views that I might have accidentally edited.